### PR TITLE
docs: improve section on prCreation

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Improve section on `prCreation`

## Context:

The current text contains some bad grammar and spelling, and was a bit hard to understand.
So I'm trying my hand at rewriting it. 😄 

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -1659,10 +1659,16 @@ This setting tells Renovate when you would like it to raise PRs:
 - `not-pending`: Renovate will wait until status checks have completed (passed or failed) before raising the PR
 - `status-success`: Renovate won't raise PRs unless tests pass
 
-Renovate defaults to `immediate` but some like to change to `not-pending`.
-If you configure to immediate, it means you will usually get GitHub notifications that a new PR is available but if you view it immediately then it will still have "pending" tests so you can't take any action.
-With `not-pending`, it means that when you receive the PR notification, you can see if it passed or failed and take action immediately.
-Therefore you can customise this setting if you wish to be notified a little later in order to reduce "noise".
+Renovate defaults to `immediate` but you might want to change this to `not-pending` instead.
+
+With prCreation set to `immediate`, you'll get a GitHub notification right away when a new Renovate bot PR is avaiable.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:1664]:
```suggestion
With prCreation set to `immediate`, you'll get a GitHub notification right away when a new Renovate bot PR is available.
```

File: docs/usage/configuration-options.md [hunk 2]
```diff
@@ -1659,10 +1659,16 @@ This setting tells Renovate when you would like it to raise PRs:
...
+Your test suite takes a bit of time to complete, so if you go look at the new PR right away, you don't know if your tests pass or fail.
+You're basically waiting until you have the test results, before you can decide if you want to merge the PR or not.
+
+With prCreation set to `not-pending`, Renovate waits untill all test have finished running, and only then creates the PR.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:1668]:
```suggestion
With prCreation set to `not-pending`, Renovate waits until all test have finished running, and only then creates the PR.
```

**Review by HonkingGoose — COMMENTED** [review]
Fixing up some typos

File: docs/usage/configuration-options.md [hunk 3]
```diff
@@ -1659,10 +1659,16 @@ This setting tells Renovate when you would like it to raise PRs:
 - `not-pending`: Renovate will wait until status checks have completed (passed or failed) before raising the PR
 - `status-success`: Renovate won't raise PRs unless tests pass
 
-Renovate defaults to `immediate` but some like to change to `not-pending`.
-If you configure to immediate, it means you will usually get GitHub notifications that a new PR is available but if you view it immediately then it will still have "pending" tests so you can't take any action.
-With `not-pending`, it means that when you receive the PR notification, you can see if it passed or failed and take action immediately.
-Therefore you can customise this setting if you wish to be notified a little later in order to reduce "noise".
+Renovate defaults to `immediate` but you might want to change this to `not-pending` instead.
+
+With prCreation set to `immediate`, you'll get a GitHub notification right away when a new Renovate bot PR is available.
```
Comment (rarkins) [inline docs/usage/configuration-options.md:1664]:
```suggestion
With prCreation set to `immediate`, you'll get a Pull Request and possible associated notification right away when a new update is available.
```

**Review by rarkins — CHANGES_REQUESTED** [review]

**Review by rarkins — DISMISSED** [review]

File: docs/usage/configuration-options.md [hunk 4]
```diff
@@ -1659,10 +1659,16 @@ This setting tells Renovate when you would like it to raise PRs:
...
-If you configure to immediate, it means you will usually get GitHub notifications that a new PR is available but if you view it immediately then it will still have "pending" tests so you can't take any action.
-With `not-pending`, it means that when you receive the PR notification, you can see if it passed or failed and take action immediately.
-Therefore you can customise this setting if you wish to be notified a little later in order to reduce "noise".
+Renovate defaults to `immediate` but you might want to change this to `not-pending` instead.
+
+With prCreation set to `immediate`, you'll get a Pull Request and possible associated notification right away when a new update is available.
+Your test suite takes a bit of time to complete, so if you go look at the new PR right away, you don't know if your tests pass or fail.
+You're basically waiting until you have the test results, before you can decide if you want to merge the PR or not.
+
+With prCreation set to `not-pending`, Renovate waits until all test have finished running, and only then creates the PR.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:1668]:
```suggestion
With prCreation set to `not-pending`, Renovate waits until all tests have finished running, and only then creates the PR.
```

Add missing `s`.

**Comment by HonkingGoose** [comment]
Sorry, I noticed a spelling error. 🙈

**Review by rarkins — APPROVED** [review]

**Pull request merged**